cypress.json

```diff
@@ -0,0 +1,11 @@
+{
+  "baseUrl": "http://localhost:3000",
+  "testFiles": ["**/*.spec.ts"],
```

Comment:
Then you will use `**/*.spec.ts` for cypress and `**/*.test.ts` for unit test, right?

Reply:
Yes @hoangmirs 
- the `**/*.test.ts ` is for unit test (Jest) and it'll be on the same directory as the code that it test
- the `**/*.spec.ts` is for UI test (Cypress) and it'll be in `cypress` folder as the test would cover more than one component or element on the codebase